DOCS-559 Self-service checkout (#4778)
SumoLogic/sumologic-documentation: docs/get-started/sign-up.md

1. In the email, click **Activate Now**.
1. Fill out the **Activate Your Account** form, then click **Activate**.

Thanks for signing up! Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with your data in a couple of minutes.

### Set up Collector

1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />

> 1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the OpenTelemetry collector and relevant dashboards.

Reviewer: they can also skip this step by going to the hamburger menu in the top right and choosing to skip.

1. Open the email and click **Activate Now**.
1. Fill out the **Activate Your Account** form, then click **Activate**.

Thanks for signing up! Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with your data in a couple of minutes.

### Set up Collector

1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />
1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the OpenTelemetry collector and relevant dashboards.

> 1. When finished, click **Start using Sumo**. <br/><img src={useBaseUrl('img/get-started/install-otel.png')} alt="icon" style={{border: '1px solid gray'}} width="500" />

Author: ```suggestion
```

1. Open the email and click **Activate Now**.
1. Fill out the **Activate Your Account** form, then click **Activate**.

Thanks for signing up! Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with your data in a couple of minutes.

### Set up Collector

1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />
1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the OpenTelemetry collector and relevant dashboards.
1. When finished, click **Start using Sumo**. <br/><img src={useBaseUrl('img/get-started/install-otel.png')} alt="icon" style={{border: '1px solid gray'}} width="500" />

> Next, you'll be taken to your onboarding checklist guide, where you can view your Dashboards, run your first log search, set up alerts, install more apps for your environment, and more.

Author: ```suggestion
Next, you'll be taken to your onboarding checklist guide, where you can view dashboards, run your first log search, set up alerts, install more apps for your environment, and more.
```

### Set up Collector

> 1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />
> 1. Copy, paste, and run the provided command in your Terminal or PowerShell. This will install the OpenTelemetry collector and relevant dashboards. When the installation is complete, click **Start using Sumo**.

Author: ```suggestion
1. Select the platform (Linux, Windows, macOS, or AWS) from which you want to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />
1. Copy, paste, and run the provided command in your Terminal or PowerShell. This will install the OpenTelemetry collector and relevant dashboards. When the installation is complete, click **Start using Sumo**. <br/><img src={useBaseUrl('img/get-started/install-otel.png')} alt="icon" style={{border: '1px solid gray'}} width="500" />
```

<sup>3</sup> Forwards your logs to an S3 bucket after being collected and analyzed in Sumo Logic.<br/>
<sup>4</sup> Notifies you in real-time when errors occur on your systems.

>  To unlock premium features and scale your Sumo Logic usage as your needs grow, upgrade to a paid plan. For more information, see [Sumo Logic Pricing](https://www.sumologic.com/pricing) and [Managing Your Subscription](/docs/manage/manage-subscription).

Reviewer 2: ```suggestion
 To unlock premium features and scale your Sumo Logic usage as your needs grow, upgrade to a paid plan. For more information, see [Sumo Logic Pricing](https://www.sumologic.com/pricing) and [Manage Subscription](/docs/manage/manage-subscription).
```

### Upgrade to a Flex plan

Using our service checkout, you can quickly and easily sign up for our Flex Essentials plan with a credit card in just minutes.  

> The Flex Essentials plan is ideal for:  

Reviewer: Is flex essentials plan referring to this - https://www.sumologic.com/pricing/?igaag=144244772630&igaat=&igacm=18138484988&igacr=703522989243&igakw=sumo%20logic%20pricing&igamt=e&igant=g

Author: I took it from the FAQ (_We are introducing a Self-Checkout Page to allow users to purchase Sumo Logic plans directly through the product interface. Customers who are in free and trial plan type will be able to upgrade to **flex essentials** easily_.)

1. Here, you can review the available Sumo Logic plans and pricing. Select the **Flex Essentials** plan.
1. Enter your credit card details in the secure checkout form.  
1. Review and confirm your purchase.  
1. Once your payment has been processed, your account will be upgraded, and you’ll have immediate access to premium features.  

For information on updating your payment information, see [Manage Billing Information](/docs/manage/manage-subscription/manage-billing-information).

:::note limitations
At this time:  
- Customers using order forms for billing are not eligible to switch to self-service payments.  
- Downgrades to the free plan cannot be completed through the checkout interface.  
- Downgrades between paid plans take effect at the start of the next billing cycle.  
:::

If you encounter any issues during the upgrade process, contact our [support team](https://support.sumologic.com/support/s/).  

### Upgrade to an enterprise plan

> Contact our [Sales Team](https://www.sumologic.com/contact-us/) for assistance. We'll consult with you to determine the best solution for your organization.

Reviewer: I don't see any of the new plans mentioned. see picture below. 
![Screenshot 2024-11-25 at 9 15 20 AM](https://github.com/user-attachments/assets/9d31dc13-d5ba-4464-973d-f120ffa9d7eb)

Author: Thank you! Please see the updated doc [here](https://github.com/SumoLogic/sumologic-documentation/pull/4778/files).

## Resources

> * [Sumo Logic quickstart](/docs/get-started/quickstart)

Author: ```suggestion
* [Sumo Logic Quickstart](/docs/get-started/quickstart)
```

* [Sumo Logic Quickstart](/docs/get-started/quickstart)
* [Onboarding checklists](/docs/get-started/onboarding-checklists)

> * [Sumo Logic training and certifications](/docs/get-started/training-certification-faq)

Author: ```suggestion
* [Free Sumo Logic Training and Certification](/docs/get-started/training-certification-faq)
```
